src/compressed_tensors/quantization/quant_args.py

```diff
@@ -212,6 +212,17 @@ def pytorch_dtype(self) -> torch.dtype:
         else:
             raise ValueError(f"Invalid quantization type {self.type}")
 
+    def remove_defaults(self):
+        defaults = self.__fields__
+        result = {}
+        for field, value in dict(self).items():
+            if field == "observer_kwargs" and len(value) == 0:
```

Comment:
Why can't we compare directly to the default value? This is inflexible if we want to change the kwargs to default to a non-empty dict